sprockets/http/app.py

```diff
@@ -120,6 +130,10 @@ def stop(self, io_loop):
         for callback in self.on_shutdown_callbacks:
             try:
                 maybe_future = callback(self.tornado_application)
+
+                if ASYNCIO_TORNADO and asyncio.iscoroutine(maybe_future):
+                        maybe_future = asyncio.create_task(maybe_future)
+
```

Comment:
Sorry, I am not well versed in Tornado's concurrent module. In an effort to learn, I have a question. In the new conditional block, would it make sense to set running_async = True? Because we know that maybe_future is a coroutine, does it follow that it is also "future"?